packages/ra-core/src/controller/useListParams.spec.tsx

```diff
@@ -182,4 +189,144 @@ describe('useListParams', () => {
             expect(result).toEqual(0);
         });
     });
+    describe('showFilter', () => {
+        it('should initialize displayed filters', () => {
+            const TestedComponent = () => {
+                const [_, { showFilter }] = useListParams({
```

Comment:
You don't need the `_`, just leave the slot it empty
```suggestion
                const [, { showFilter }] = useListParams({
```